src/hooks/useENSModifiedRegistration.ts

```diff
@@ -89,6 +89,8 @@ export default function useENSModifiedRegistration({
         ...profileQuery.data?.records,
         ...profileQuery.data?.coinAddresses,
       } as Records;
+      // delete ETH since we don't show it in the ENS form
+      delete initialRecords.ETH;
```

Comment:
deleting ETH from `initialRecords` we don't have that label in the form so the form was empty at the beginning